modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -106,12 +106,12 @@ If mcrypt is not available, either a pure-PHP implementation or OpenSSL is used
 The prioritization is as follows: OpenSSL > mcrypt > pure-PHP. 
 mcrypt and OpenSSL load faster than the pure-PHP implementation. 
 mcrypt offers a 45x speedup over pure-PHP.
-OpenSSL offers a 6.5x speedup over mcrypt.
+OpenSSL offers a more than 6.5x speedup over mcrypt.
```

Comment:
"greater than" is correct in this case as you're referring to an uncountable item.